apps/src/maker/ui/SetupGuide.jsx

```diff
@@ -39,8 +40,18 @@ export default class SetupGuide extends React.Component {
             dismissible
           />
         )}
+
         <h1>{applabI18n.makerSetupPageTitle()}</h1>
 
+        <Alert
+          text={applabI18n.chrome133PlusBugCallout()}
+          link={{
+            href: 'https://status.code.org/cmb0s8jbr002zpulbal0ei011',
```

Comment:
Out of curiosity, do we know if this is a hard link? Or at least do we feel this link will stay persistent while this issue is ongoing?

Reply:
@bethanyaconnor - do you mind weighing on on this question? Thanks!
We could provide a link to the [support article](https://support.code.org/hc/en-us/articles/9953258694669-Connecting-the-Circuit-Playground-or-micro-bit-to-Your-Browser-A-Step-by-Step-Guide) instead which includes the announcement.

Reply:
I think it should be, but not sure off the top of my head. That said, when we do eventually close out the status, it will disappear within a couple days, so I think either directing to status.code.org or to the support article makes more sense.